clarity_ext/service/clarity_service.py

```diff
@@ -52,24 +52,24 @@ def update(self, domain_objects, ignore_commit=False):
     def _update_process(self, process):
         # Updates the process itself. Currently only the udfs
         for item in process.udf_map.enumerate_updated():
-            print(item.key, item.value)
+            print((item.key, item.value))
```

Comment:
This print statement must be there by accident, so we could also just remove it.